compiler/tests-num/dune

```diff
@@ -1,31 +1,18 @@
-(executable
+(test
  (name main)
  (modules main test_nats test test_big_ints test_ratios test_nums test_io)
  (libraries num)
+ (enabled_if %{env:js-enabled=})
  (modes
   js
   (best exe))
  (flags
   (:standard -linkall -w -3-7-33-35-37 -safe-string -no-strict-sequence)))
 
 (rule
- (target main.referencejs)
- (deps main.bc.js)
- (action
-  (with-stdout-to
-   %{target}
-   (run node ./main.bc.js))))
-
-(rule
- (target main.reference)
+ (target main.expected)
```

Comment:
Do we need to keep this rule ?

Reply:
This produces the file `main.expected`, which is then used as a reference when testing the generated JavaScript code. But one could commit this file instead.

Reply:
Let's keep it then